### DISCUSSION TITLE
TP-Link T2UB Nano (Realtek RTL8821CU)

### DISCUSSION THREAD

**Discussion by aljims** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#359] [Show and tell]
Wifi on 2.4GHz and 5GHz is working! This adapter has wifi and bluetooth combo (RTL8821CU). Only the wifi works.

![Screenshot 2024-09-28 at 3 02 35 AM](https://github.com/user-attachments/assets/20fe600c-5a18-4eba-9347-df5a1118bcb0)

**Comment by chris1111** [comment]
Thanks 
Bluetooth not working on macOS you need compatible dongle Bluetooth like this [KINIVO](https://www.amazon.ca/Kinivo-Bluetooth-Wireless-Receiver-Raspberry/dp/B0BQYL2PK3) With [BlueToolFixup.kext](https://github.com/acidanthera/BrcmPatchRAM/releases)

**Reply by aljims** [reply]
Got it. Thanks!

**Reply by aljims** [reply]
Btw, can I ask something? Can I disable the PBC on startup and just let the app connect to wifi? Or it's just the way the app works in order to make wifi work?

**Comment by chris1111** [comment]
The Utility is suppose to connect a known wifi auto with out using PBC
